### PR TITLE
New package: OpenDoas-0.1

### PR DESCRIPTION
**Pull request by Duncaen**
@chneukirchen

### CODE REVIEW

**Comment by Vaelatern** [comment]
Can we have a lowercase name? Since the uppercase names are so confusing to find...

File: srcpkgs/OpenDoas/INSTALL [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#
+# This script sets up correct perms for usr/bin/doas.
+#
+case "${ACTION}" in
+post)
+	chmod 4111 usr/bin/doas
```
Comment (Vaelatern) [inline srcpkgs/OpenDoas/INSTALL]: Can't be done in `post_install()`?

Comment (Duncaen) [inline srcpkgs/OpenDoas/INSTALL]: afaik tar would drop the suid bit, sudo etc. use the `INSTALL` script too.

**Comment by Duncaen** [comment]
The standard is using the upstream name, there is doas in small in the short_description, so it matches lowercase too.

File: srcpkgs/OpenDoas/INSTALL (hunk 1 above)
Comment (Vaelatern) [inline srcpkgs/OpenDoas/INSTALL]: Please see `pkgname=iputils` line 29

Comment (Vaelatern) [inline srcpkgs/OpenDoas/INSTALL]: I know this example because I broke it in the libressl version bump.

File: srcpkgs/opendoas/files/doas.pam [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#%PAM-1.0
```
Comment (chneukirchen) [inline srcpkgs/opendoas/files/doas.pam]: Perhaps add an commented example for pam_timestamp here?

Comment (Duncaen) [inline srcpkgs/opendoas/files/doas.pam]: Yes why not, added it.

**Comment by Gottox** [comment]
So excited! Could this replace sudo in base?

**Pull request merged**